src/torchcrf/__init__.py

```diff
@@ -111,16 +110,16 @@ def forward(self,
                 raise ValueError('mask of the first timestep must all be on')
 
         if mask is None:
-            mask = Variable(self._new(tags.size()).fill_(1)).byte()
+            mask = torch.Tensor(self._new(tags.size()).fill_(1)).byte()
 
         numerator = self._compute_joint_llh(emissions, tags, mask)
         denominator = self._compute_log_partition_function(emissions, mask)
         llh = numerator - denominator
         return llh if not reduce else torch.sum(llh)
 
     def decode(self,
-               emissions: Union[Variable, torch.FloatTensor],
-               mask: Optional[Union[Variable, torch.ByteTensor]] = None) -> List[List[int]]:
+               emissions: Union[torch.Tensor, torch.FloatTensor],
+               mask: Optional[Union[torch.Tensor, torch.ByteTensor]] = None) -> List[List[int]]:
```

Comment:
Since now variable and tensor are the same, emissions should be just `torch.FloatTensor` and mask should be just `torch.ByteTensor`. No need for `Union`.